packages/embed/src/browser.ts

```diff
@@ -4,11 +4,19 @@ import {
   initializeSidetabs,
   initializeSliders,
   initializeWidgets,
+  initializeSingleEmbeds,
 } from './initializers'
 
 import * as lib from './index'
 
 function loadEmbedElements(forceReload: boolean = false) {
+  initializeEmbedElements(forceReload)
```

Comment:
I think this first line is redundant, since it will be called again in line 15 regardless of whether there were any single embeds or not. So I would delete it, wdyt?

Reply:
I thought similar but then I had a little [discussion](https://typeform.slack.com/archives/C054LLXFMGS/p1684853699724279) with @mathio and the suggestion was to first load the non-single-code embeds and then fetch the single-code ones to not block the regular ones. What's your opinion on this? 🤔